### PR TITLE
Added content of 15MMM Toolkit to activities.

### PR DESCRIPTION
**Pull request by gandhiano**
I think it would work better to put 15MMM as a tab in the site, separate page from activities (which tend to be updated?). But as I don't have time to explore the source structure, here is a first increment on the documentation.

Some buttons to replace the links would also be good in a next improvement, if someone has time available for it.

### CODE REVIEW

**Comment by almereyda** [comment]
The navigation bar is a seperate issue.
Let me check how it looks like, then let's decide on certain criteria to meet for this PR before it can be merged. Like:
- [x] Table CSS

Else it shouldn't be merged + deployed. We could then tomorrow also check
- [ ] the contents.
- [ ] Buttons?
- [x] Navbar?
- [x] Logos?

**Comment by gandhiano** [comment]
I suggest you leave the criteria as simple as possible. Contents should be checked, but we should not let information be blocked because of design details, especially if there's not someone immediately volunteering to deal with these. 

They can always be improved by a later PR, but the priority now is to have simple and clear information out.

Deploy early, deploy often.

File: src/partials/text-15mmm.html.md [hunk 1]
```diff
@@ -1,14 +1,47 @@
...
+| [Semantic Mediawiki](http://mmm.3oe.de/wiki/Main_Page) | [OpenStreetMap id](http://editor.transformap.co) |
+| --- | --- |
+| *form based* | *map based* |
+
+### Feedback
+We welcome all feedback on TransforMap, the tools and the taxonomy. This feedback is what helps us gear our development towards the interest of the growing number of communities involved.
+
+Clicking the links below will open your e-mail program to submit questions or issues to the project developers.
+
+* [Ask a question](mailto:questions@transformap.co)
```
Comment (almereyda) [inline src/partials/text-15mmm.html.md:29]: `s/questions/mapping/` as of my last comment on https://trello.com/c/ShbWsauB/11-make-assesment-of-helpdesk-tools-for-15mmm-and-chest-aftwords

File: src/partials/text-15mmm.html.md [hunk 2]
```diff
@@ -1,14 +1,47 @@
...
+* [Submit an issue](mailto:issues%40transformap.co?subject=REPLACE%20with%20your%20issue%20subject&body=%3E%20Please%20describe%20your%20issue%20below%0A%0A%0A%3E%20If%20you%20want%20to%20get%20contacted%20by%20us%2C%20write%20your%20e-mail%20address%20below%3A%0Ayourmail%40yourserver.org%0A%0A%3E%20You%20can%20follow%20your%20and%20other%20submitted%20issues%20by%20visiting%20https%3A%2F%2Ftrello.com%2Fb%2F9R9eccaK%2F15mmm)
```
Comment (almereyda) [inline src/partials/text-15mmm.html.md:30]: `s/issues/questions/` same reason as above

File: src/layouts/default.html.eco [hunk 3]
```diff
@@ -64,7 +64,7 @@
 					<% end %>
 					<li>
 						<a href="//discourse.transformap.co">
-							Forum
+							Agora
```
Comment (almereyda) [inline src/layouts/default.html.eco:67]: This needs to be changed to **Discourse** or **Community** (at will) in following the unrepresentative survey at http://discourse.transformap.co/t/domain-name-for-this-discourse-forum/383/11

Forum is just out, sorry.

**Comment by almereyda** [comment]
Logos will have to be part of a subsequent PR based on the discussion in http://discourse.transformap.co/t/collecting-supporter-and-partner-logos/365/78 and the data in https://github.com/TransforMap/transformap-partners

**Comment by almereyda** [comment]
I know we should be merging more often ... http://martinfowler.com/bliki/MonolithFirst.html ... Practicality beats Purity ... deploy early, deploy often ... but I want the _landing page_ to stay beautiful and not grow bewilderingly. Any opinions on that appreciated.

Also see this ;)

dashboard_aller_webseiten_-_piwik_ _webanalytik_berichte_-_2015-06-04_01 34 20

admin_-_transformap_-_2015-06-04_01 37 37

---

Meaning one third of all our Discourse' posts have been written last month and probably one sixth of our landing page visitors end up in the Discourse somehow. Then, supposedly, they can get lost ;)

File: src/layouts/default.html.eco (hunk 3 above)
Comment (species) [inline src/layouts/default.html.eco:67]: There was a 4:4:4 EQUAL vote for Forum, Discourse and Community. Forum isn't out, so who is about to decide?

Comment (almereyda) [inline src/layouts/default.html.eco:67]: I'm using my veto against Forum. Maybe we should escalate this question to Loomio or DemocracyOS.

**Comment by almereyda** [comment]
Maybe @gandhiano is right and we turn 15MMM into a seperate page? But soon we'll get many. Also the global nav should be reserved for generalized services, not every little thing we find important.

Is it time to think about a sub navigation? additionally to pop ups?

**Comment by almereyda** [comment]
?

File: src/layouts/default.html.eco (hunk 3 above)
Comment (almereyda) [inline src/layouts/default.html.eco:67]: http://discourse.transformap.co/c/community-building > **Communities** it is, then?

@thoka @gandhiano @species

**Comment by gandhiano** [comment]
This is out of date by now.

**Pull request closed without merging**